inscriptions/ISic000586.xml

```diff
@@ -185,14 +190,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><expan><abbr>d</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><name><expan><abbr>M</abbr><ex>arcus</ex></expan></name> <g ref="#interpunct">·</g> <name><expan><abbr>Volumni</abbr><ex>us</ex></expan></name> <name>Eratus</name>
+                    <lb n="3"/><expan><abbr>v</abbr><ex>ixit</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>a</abbr><ex>nnos</ex></expan> <g ref="#interpunct">·</g> <num value="37">XXXVII</num> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>enses</ex></expan> <g ref="#interpunct">·</g> <num value="7">VII</num>
+                    <lb n="4"/><name><expan><abbr>C</abbr><ex>aius</ex></expan></name> <g ref="#interpunct">·</g> <name>Ancitus</name> <g ref="#interpunct">·</g> <hi rend="ligature">am</hi>ico <g ref="#interpunct">·</g> <hi rend="ligature"><expan><abbr>m</abbr><ex>emoriam</ex></expan> <expan><abbr>p</abbr><ex>osuit</ex></expan></hi> <!-- I'm not sure if representing MP in ligature with expansions will work? -->
```

Comment:
a tricky one. We would (I consulted), suggest:
`<expan><abbr><hi rend="ligature" xml:id="h1"><am>m</am></hi></abbr><ex>memoriam</ex></expan>
                    <expan><abbr><hi rend="ligature" xml:id="h2"><am>p</am></hi></abbr><ex>posuit</ex></expan>
                    <link target="#h1 #h2"/>`

Reply:
An incredibly complex solution! Given that the 'm' of memoriam is in the abbreviation (abbr), should it be repeated in the expansion (ex) as you have it here? Same with 'p' of 'posuit'?